docs/sources/administration/data-source-management/index.md

```diff
@@ -4,6 +4,7 @@ aliases:
   - ../datasources/datasource_permissions/
   - ../enterprise/datasource_permissions/
   - ../enterprise/query-caching/
+  - ../enterprise/query-and-resource-caching/
```

Comment:
Did this page used to exist? Just checking that we in fact need an alias here.

Reply:
Ah okay - I didn't realize that's what aliases were for. I will remove it.

Reply:
Done [d2fdcf7](https://github.com/grafana/grafana/pull/70556/commits/d2fdcf7a9fa2854e04329ff81095b316ab8e5e6b)